website/static/js/pages/prereg-landing-page.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+var $ = require('jquery');
+var $osf = require('js/osfHelpers');
+var Raven = require('raven-js');
+var ko = require('knockout');
+require('js/qToggle');
+require('js/onboarder.js');
+require('js/components/autocomplete');
+
+$(function(){
+    $('.prereg-button').qToggle();
+    $('.prereg-button').click(function(){
+        var target = $(this).attr('data-qToggle-target');
+        var input = $(target).find('input').first().focus();
+    });
+
+    $('#newProject, #newProjectXS').click( function() {
+        var title = $(this).parent().find('.new-project-title').val();
+        if (!title) {
+            return;
+        }
+        $osf.postJSON('/api/v1/project/new/', {
+            title: title,
+            campaign: 'prereg'
+        }).done(function(response) {
+            window.location = response.projectUrl + 'registrations/?c=prereg';
```

Comment:
This link (and possibly others in the diff) may have been broken by the change to "campaign" as the name of the URL parameter.